rest_framework_nested/serializers.py

```diff
@@ -27,10 +27,10 @@ class NestedHyperlinkedModelSerializer(rest_framework.serializers.HyperlinkedMod
 
     def __init__(self, *args, **kwargs):
         self.parent_lookup_kwargs = kwargs.pop('parent_lookup_kwargs', self.parent_lookup_kwargs)
-        super(NestedHyperlinkedModelSerializer, self).__init__(*args, **kwargs)
+        super().__init__(*args, **kwargs)
 
     def build_url_field(self, field_name, model_class):
-        field_class, field_kwargs = super(NestedHyperlinkedModelSerializer, self).build_url_field(
+        field_class, field_kwargs = super().build_url_field(
```

Comment:
That part can stay. It helps typing tools.
```suggestion
        field_class, field_kwargs = super(NestedHyperlinkedModelSerializer, self).build_url_field(
```